lms/djangoapps/bulk_email/models.py

```diff
@@ -91,7 +92,12 @@ def create(cls, course_id, sender, to_option, subject, html_message, text_messag
             text_message = html_to_text(html_message)
 
         # perform some validation here:
-        if to_option not in TO_OPTIONS:
+        if to_option.isdigit():
+            if not GroupedQuery.objects.filter(id=int(to_option)).exists():
+                fmt = 'Course email for "{course}" being sent to query id that does not exist: {query_id}, subject "{subject}"'
```

Comment:
I'm not a fan of truncated variable names. This culture was largely born in a world were there actually were limits placed on the lengths of identifiers.

Since we have no such constraints, I tend to opt for verbose/explicit identifiers, like `format` and `message`.